### PR TITLE
Release to rubygems

### PR DESCRIPTION
**Pull request by Magons**

### CODE REVIEW

**Comment by NazarKY** [comment]
@Magons pls release the new version of the gem

here is this gem https://rubygems.org/gems/oauth2-me

also change the version of gem [Software versioning](http://en.wikipedia.org/wiki/Software_versioning)

**Comment by Magons** [comment]
@NazarKY  pls [review](https://github.com/jetthoughts/oauth2-me-ruby/pull/1/files)

File: lib/oauth2-me/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module OAuth2Me
-  VERSION = '0.0.1a'
+  VERSION = '0.1.1a'
```
Comment (pftg) [inline lib/oauth2-me/version.rb]: pls remove `a` suffix

**Comment by NazarKY** [comment]
@Magons pls publish the gem to rubygems

**Comment by Magons** [comment]
Version 0.1.1 [released](https://rubygems.org/gems/oauth2-me).

**Comment by NazarKY** [comment]
+1
@pftg pls review the link above

**Pull request merged**